.stackblitz/app.vue

```diff
@@ -0,0 +1,54 @@
+<script lang="ts" setup>
+// Usage of `useCldImageUrl` composable
+const { url } = useCldImageUrl({ src: '/cld-sample-5.jpg' })
+console.log(url)
+</script>
+
+<template>
+  <!-- Usage of `CldImage.vue` component -->
+  <CldImage
+    :src="'cld-sample-5.jpg'"
```

Comment:
just curious - does it require the double quotes like that? `"'string'"` ?

Reply:
No, it does need that. I just changed it :)